src/packages/@workspaces/features/invite-to-workspace/components/invite-user-pickers/InviteUserPicker.svelte

```diff
@@ -0,0 +1,282 @@
+    const emailContainer = document.getElementById(
+      "input-email",
+    ) as HTMLElement;
+    emailContainer.removeChild(removeElement);
+    emailstoBeSentArr = emailstoBeSentArr?.filter((e) => e != email);
+    onChange(emailstoBeSentArr);
+    filterUser();
+  };
+</script>
+
+<div on:click={handleDropdownClick}>
+  <div
+    id={`input-select-list-${id}`}
+    class="parent-dropdown display-inline-block z-1 border-radius-4 {isError
+      ? 'selectErrorBorderClass'
+      : ''}
+    
+    "
+    style=" position: relative;"
+  >
+    <div>
+      <div class="email-container rounded d-flex flex-wrap">
+        <div
+          id="input-email"
+          class="d-flex align-items-start flex-wrap gap-2"
+        ></div>
+        <input
+          id="input"
+          placeholder={!emailstoBeSentArr.length
+            ? "Enter email ID or name of team member."
+            : ""}
+          autocomplete="off"
+          autocapitalize="none"
+          style="outline:none;border:none;flex-grow:1; background:transparent;"
+          bind:value={currentEmailEntered}
+          on:input={() => {
+            filterUser();
+          }}
+          on:focus={() => {
+            isOpen = true;
+          }}
+          class="input-container mt-1 sparrow-fs-12 my-1"
+        />
+      </div>
+    </div>
+
+    {#if isOpen}
+      <div
+        class="d-none select-data p-1 rounded"
+        class:select-active={isOpen}
+        transition:slide={{ duration: 100 }}
+      >
+        {#each data as user, index}
+          <div
+            class="d-flex w-100 px-2 py-2 highlight rounded"
+            on:click={() => {
+              handleEmailOnAdd(user.email);
+              isOpen = false;
+              filterUser();
+            }}
+          >
+            <div class="d-flex tile w-100 rounded align-items-center">
+              <div class="info d-flex align-items-center w-100">
+                <div
+                  class="icon d-flex align-items-center justify-content-center"
+                >
+                  <span>{user.name[0].toUpperCase()}</span>
+                </div>
+                <div class="name px-2 ellipsis">
+                  <span class="sparrow-fs-12 text-whiteColor w-100 ellipsis"
+                    >{user.name || ""}</span
+                  ><br />
+                  <span class="sparrow-fs-12 text-secondary-200 w-100 ellipsis"
+                    >{user.email}</span
+                  >
+                </div>
+              </div>
+            </div>
+          </div>
+        {/each}
+        {#if !data?.length}
+          <p class="sparrow-fs-12 m-0 ps-2 p-0 pt-1 pb-1 text-whiteColor">
+            {"No members found!"}
+          </p>
+        {/if}
+      </div>
+    {/if}
+  </div>
+</div>
+
+<style>
+  .email-container {
+    background-color: var(--bg-tertiary-300);
+    border: 1px solid;
+    padding: 3px 8px 3px 8px;
+    border: 1px solid var(--border-color);
```

Comment:
use standard format for border color like --border-primary-100

Reply:
resolved